pkg/morph/client/notary.go

```diff
@@ -496,199 +530,85 @@ func (c *Client) notaryInvoke(committee, invokedByAlpha bool, contract util.Uint
 		}
 	}
 
-	// prepare main tx
-	mainTx := &transaction.Transaction{
-		Nonce:           nonce,
-		SystemFee:       test.GasConsumed,
-		ValidUntilBlock: until,
-		Script:          test.Script,
-		Attributes: []transaction.Attribute{
-			{
-				Type:  transaction.NotaryAssistedT,
-				Value: &transaction.NotaryAssisted{NKeys: u8n},
-			},
-		},
-		Signers: cosigners,
-	}
-
-	// calculate notary fee
-	notaryFee, err := c.client.CalculateNotaryFee(u8n)
+	cosigners, err := c.notaryCosigners(invokedByAlpha, alphabetList, committee)
 	if err != nil {
 		return err
 	}
 
-	// add network fee for cosigners
-	//nolint:staticcheck // waits for neo-go v0.99.3 with notary actors
-	err = c.client.AddNetworkFee(
-		mainTx,
-		notaryFee,
-		c.notaryAccounts(invokedByAlpha, multiaddrAccount)...,
-	)
+	nAct, err := notary.NewActor(c.client, cosigners, c.acc)
```

Comment:
Again, this actor can and should be used long-term, not recreated for every request.

Reply:
Then we have to track the committee updates somehow to recreate the actor on update.

Reply:
This should be done anyway. If it's not, well, we need an issue.

Reply:
Added an issue, it's kind of a separate feature which requires some more attention to carefully watch the committee updates and keep Notary actor in sync with it.